opentelemetry-sdk/tests/logs/test_handler.py

```diff
@@ -79,6 +79,12 @@ def test_log_record_emit_noop(self):
             logger.warning("Warning message")
         handler_mock._translate.assert_not_called()
 
+    def test_log_flush_noop(self):
```

Comment:
Looks like we need to add a test for the new behavior here (calling `force_flush` on the `LoggingHandler`), no?